app/src/main/java/org/p2p/wallet/user/interactor/TokenMetadataInteractor.kt

```diff
@@ -10,7 +10,7 @@ import org.p2p.wallet.common.storage.ExternalStorageRepository
 import org.p2p.wallet.user.repository.UserLocalRepository
 
 private const val TAG = "TokenMetadataInteractor"
-private const val TOKENS_FILE_NAME = "tokens"
+private const val TOKENS_FILE_NAME = "tokens_metadata"
```

Comment:
I assume that versioning could be a good idea, otherwise if something changes we will need to come up with a new value. What do you think?
```suggestion
private const val TOKENS_FILE_NAME = "tokens_metadata_v1"
```

Reply:
I think there is no need to do thiы